docs/operations/runbooks/downtime_incident_management.md

```diff
@@ -0,0 +1,29 @@
+# Downtime Incident Management Runbook
+
+ Our team has agreed upon steps for handling incidents that cause our site to go offline or become unusable for users. For this document, an incident refers to one in which manage.get.gov is offline or displaying error 400/500 HTTP errors on all pages and is caused by a critical bug in our code, not to be confused with a security incident. This document should not be used for security incident response.
+
+## Response management rules
+
+The following set of rules should be followed while an incident is in progress.
+
+- The person who first notices that the site is down is responsible for using @here and notifying in #dotgov-announce that production is down
```

Comment:
Should this be the new @dotgov-redalert slack channel?

Reply:
No not everyone is in dotgov-redalert cahnnel. This is to make sure everyone is in the loop